cmd/soroban-cli/src/commands/config/alias.rs

```diff
@@ -1,103 +1,8 @@
-use std::{
-    collections::HashMap,
-    fs::{self, create_dir_all, OpenOptions},
-    io::Write,
-    path::PathBuf,
-};
+use std::collections::HashMap;
 
 use serde::{Deserialize, Serialize};
-use stellar_strkey::DecodeError;
-
-use crate::commands::config;
-
-use super::Args;
 
 #[derive(Serialize, Deserialize, Default)]
 pub struct Data {
```

Comment:
At this point you could replace this struct with a type alias `type Alias = HashMap<String, String>` and move it to `config/locator.rs`. This type is already serializable and you won't need to `data.ids` anymore.

Reply:
Could also call it `AliasMap`?

Reply:
The idea is being open for extensions later on. If we move keys to the root not, that's not possible.